examples/nextjs-spa/pages/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react"
+import Link from "next/link"
+import "@ory/elements/style.css"
+
+// React
+import { useEffect, useState } from "react"
+import { useRouter } from "next/router"
+
+// Ory SDK
+import { ory } from "../components/sdk"
+
+// import css
+import styles from "../styles/Dashboard.module.css"
+import "@ory/elements/style.css"
```

Comment:
I think you can move this to `_app.tsx`

Reply:
IDE didn't like that

Reply:
I believe you can import `@ory/elements/style.css` inside `_app.tsx`.
See here https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules